src/modules/producer.js

```diff
@@ -49,12 +49,16 @@ class Producer {
       const responsePromise = rpcQueue[correlationId];
 
       if (responsePromise === undefined) {
+        const error = new Error(`Receiving RPC message from previous session: callback no more in memory. ${queue}`);
         this._connection.config.transport.error(
           loggerAlias,
-          new Error(
-            `Receiving RPC message from previous session: callback no more in memory. ${queue}`
-          )
+          error
         );
+        this._connection.config.logger.error({
```

Comment:
think it should be a warning, when callback is no more in memory it most of the time means a timeout..